pkg/tools/spanhelper/span_helper.go

```diff
@@ -108,6 +108,7 @@ type SpanHelper interface {
 	LogObject(attribute string, value interface{})
 	LogValue(attribute string, value interface{})
 	LogError(err error)
+	LogErrorf(format string, err error)
```

Comment:
Not sure about this API. Because we impose a restriction on the format string that it should have only one format arg. It can be not clear for users.
Probably we can use something like this:
```
LogErrorf(msg string, err error) 
```